src/test/java/String/StringTest.java

```diff
@@ -0,0 +1,76 @@
+package String;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+  @Test
+  public void splitTest1() {
+    //given
+    String string = "1,2";
+
+    //when
+    String[] split = string.split(",");
+
+    //then
+    assertThat(split).contains("1", "2");
+  }
+
+  @Test
+  public void splitTest2() {
+    //given
+    String string = "1";
+
+    //when
+    String[] split = string.split(",");
+
+    //then
+    assertThat(split).containsExactly("1");
+  }
+
+  @Test
+  public void subStringTest() {
+    //given
+    String string = "(1,2)";
+
+    //when
+    String result = string.substring(1, string.length() - 1);
+
+    //then
+    assertThat(result).isEqualTo("1,2");
+  }
+
+  @Test
+  @DisplayName("특정위치의 문자열 가져오기")
```

Comment:
테스트를 설명할 때는 행위만 명시하기보단 이 테스트가 어떤상황에서 어떤 결과를 보여줄지에 대해 작성하는게 좋습니다. 

```suggestion
  @DisplayName("문자열 범위내의 위치한 문자를 가져오려 할 경우, 해당 위치의 문자를 가져온다.")
```

Reply:
@catsbi 
"어떤 것을 테스트 하고 그 결과값은 무엇이다"를 조금 더 상세하게 표현해야 겠네요.